### ISSUE TITLE
Auto refresh rooms list / homepage periodically

### ISSUE THREAD

**Issue by andrewthecoder** [pdaddyo/soundbounce#281]
Just noticed today that the homepage showed no users in Liquifaction, then when I entered there were 5 users.

I realized I'd opened the client about 2 hours earlier (8am) and I'm guessing it currently doesn't auto-refresh.

Either way, if we're not doing this yet we should be.

**Comment by pdaddyo** [comment]
Good point. We can auto-update the listener counts for sure, but shouldn't re-order based on the listener counts, because then the button under your mouse could change just before you click which is super annoying.
